docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       args:
         DOCKER_USER: ${DOCKER_USER:-1000}
     user: ${DOCKER_USER:-1000}
-    image: ralph:development
+    image: fundocker/ralph:development
```

Comment:
interesting, we never do that... why did you choose to change it ? It's just to ease the use in the CI ?

Reply:
To ease locally build image publication via: `docker-compose push`